docs/widgets/tool_tip.mdx

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Can you please add this doc to `docs.json` under the widgets tab? Otherwise it won't show on the doc. 

You can also text the doc using [mintlify CLI](https://www.mintlify.com/docs/installation) and running `mint dev` locally.